src/tests/relay/data/texample_sidre_basic_ds_demo.sidre_conduit_json

```diff
@@ -10,13 +10,45 @@
         {
           "i64": 
           {
-            "state": {"dtype":"char8_str", "number_of_elements": 7, "offset": 0, "stride": 1, "element_bytes": 1, "endianness": "little", "value": "SCALAR"},
-            "value": {"dtype":"int64", "number_of_elements": 1, "offset": 0, "stride": 8, "element_bytes": 8, "endianness": "little", "value": 1}
+            "state": 
```

Comment:
I'm not sure why the formatting changed here.  Please see the `sidre_json` file for a better view of the diffs.

Reply:
I linted the files to make it easier to read, but can back out if you don't like it